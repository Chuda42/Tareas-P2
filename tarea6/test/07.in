# Prueba de operaciones de colaDePrioridad
rangoCP
estaVaciaCP
estaEnCP 5
insertarEnCP 5 1.5
estaEnCP 5
prioridad 5
estaVaciaCP
prioritario
eliminarPrioritario
estaVaciaCP
insertarEnCP 5 1.9
prioridad 5
actualizarEnCP 5 1.5
prioridad 5
prioritario
eliminarPrioritario
estaVaciaCP
insertarEnCP 5 1.5
insertarEnCP 1 1.2
prioritario
insertarEnCP 2 1.3
insertarEnCP 9 1.4
insertarEnCP 8 1.5
insertarEnCP 7 1.6
insertarEnCP 6 1.7
insertarEnCP 4 1.8
insertarEnCP 3 1.9
insertarEnCP 10 -2.5
prioritario
Fin
