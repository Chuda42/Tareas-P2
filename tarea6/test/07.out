1>#  Prueba de operaciones de colaDePrioridad.
2>10.
3>Vacia.
4>No está.
5>Se insertó el elemento.
6>Está.
7>1.50.
8>No vacía.
9>5
10>Eliminado.
11>Vacia.
12>Se insertó el elemento.
13>1.90.
14>Se actualizó el valor.
15>1.50.
16>5
17>Eliminado.
18>Vacia.
19>Se insertó el elemento.
20>Se insertó el elemento.
21>1
22>Se insertó el elemento.
23>Se insertó el elemento.
24>Se insertó el elemento.
25>Se insertó el elemento.
26>Se insertó el elemento.
27>Se insertó el elemento.
28>Se insertó el elemento.
29>Se insertó el elemento.
30>10
31>Fin.
